src/Microsoft.AspNet.Mvc.ModelBinding/Validation/AssociatedValidatorProvider.cs

```diff
@@ -1,8 +1,10 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNet.Mvc.ModelBinding.Internal;
```

Comment:
FYI System namespaces always come first.  you may need to change VS setting Tools / Options / "Text Editor" / C# / Advanced / "Place 'System' directives first when sorting usings".